app/models/custom_button_set.rb

```diff
@@ -25,4 +25,13 @@ def self.applies_to_instance(class_name, id)
   def self.applies_to_all_instances(class_name)
     applies_to_class(class_name).where("set_data not like ?", "%:applies_to_id%")
   end
+
+  def deep_copy(options = {})
```

Comment:
Since `options[:owner]` is important or mandatory, should it be a separate parameter, or document it is required in the `options`?